_includes/cohort-loop.html

```diff
@@ -0,0 +1,42 @@
+{% assign all_participants = '' %}
+{%- assign all_mentors = '' -%}
+{%- assign all_experts = '' -%}
+{%- assign all_facilitators = '' -%}
+{%- assign all_projects = '' -%}
+{%- assign cohorts = site.data.cohorts | sort %}
+
+{%- for cohort_folder in cohorts -%}
```

Comment:
Could you use the same structure of the loop (variables, etc) than in the `_layouts/people.html`?

Reply:
I made some changes to the variable declaration in `cohort-loop.html` and `index.md`. Please confirm that it is what you had in mind. 🙏